site/shared/connectors/metaMask.md

```diff
@@ -43,8 +43,13 @@ Check out the [MetaMask SDK docs](https://docs.metamask.io/wallet/connect/3rd-pa
 
 `DappMetadata | undefined`
 
-Metadata about the dapp using the SDK, including `name`, `url`, and `iconUrl`.
-By default, `name` is set to `'wagmi'`.
+Metadata about the dapp using the SDK, including:
+
+- `name` - The name of the dapp.
+  The default is `'wagmi'`.
```

Comment:
I prefer to remove this line. The default "wagmi" value is only on a specific case, when the value `window.location.origin` is not defined. I don't want to document this edge case.